drivers/mpu9250/MPU9250.cpp

```diff
@@ -250,8 +312,26 @@ int MPU9250::mpu9250_deinit()
 	return 0;
 }
 
+int MPU9250::pinThread(int cpu)
```

Comment:
I would rather just leave the Operating System Scheduler to decide. Using the existing scheduler has the downside of running on a single CPU. Of course, you could improve that, but what's the point when you have a flexible scheduler like Linux with PREEMPT?